darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1643,12 +1643,30 @@ def _batch_collate_fn(batch: list[tuple]) -> tuple:
                 aggregated.append([sample[i] for sample in batch])
         return tuple(aggregated)
 
-    def save(self, path: Optional[str] = None) -> None:
+    def _clean(self):
+        """Returns a cleaned model, keeping only the necessary attributes for prediction."""
+        model = super()._clean()
+        # Copy from super()._clean() call __getstate__ which removes model and trainer
+        # a shallow copy is enough since we are only interested in removing pointers
+        model.model = copy.copy(self.model)  # keep the model for prediction
+        model._model_params = copy.copy(self._model_params)
+        model._model_params["pl_trainer_kwargs"] = None
+        model.trainer_params = None
```

Comment:
Maybe better to assign an empty here since when creating a new TorchForecastingModel, it will always have type dict.

```suggestion
        model.trainer_params = {}
```